src/routes/DigitalAssets/modules/digitalAssets.js

```diff
@@ -1,299 +1,139 @@
-export const SEARCH_SUCCESS = '@@digitalAssets/SEARCH_SUCCESS'
-export const SEARCH_ERROR = '@@digitalAssets/SEARCH_ERROR'
-export const SET_INVALID_FIELD = '@@digitalAssets/SET_INVALID_FIELD'
-export const CLEAN = '@@digitalAssets/CLEAN'
-
-export const init = (): { type: string } => ({
-  type: INIT,
-})
-
-export const initFinish = (): { type: string } => ({
-  type: INIT_FINISH,
-})
-
-export const open = (): { type: string } => ({
-  type: OPEN,
-})
-
-export const close = (): { type: string } => ({
-  type: CLOSE,
-})
-
-export const setAssets = (items: ?DigitalAssets): {
-  type: string,
-  payload: {
-    items: ?DigitalAssets,
-  },
-} => ({
-  type: SET_ASSETS,
-  payload: {
-    items,
-  },
-})
-
-export const setAssetsSuccess = (items: DigitalAssets): {
-  type: string,
-  payload: {
-    items: DigitalAssets,
-  },
-} => ({
-  type: SET_ASSETS_SUCCESS,
-  payload: {
-    items,
-  },
-})
-
-export const setActive = (address: Address): {
-  type: string,
-  payload: {
-    address: Address,
-  },
-} => ({
-  type: SET_ACTIVE,
-  payload: {
-    address,
-  },
-})
-
-export const setCurrent = (currentAddress: ?Address): {
-  type: string,
-  payload: {
-    currentAddress: ?Address,
-  },
-} => ({
-  type: SET_CURRENT,
-  payload: {
-    currentAddress,
-  },
-})
-
-export const getBalances = (): { type: string } => ({
-  type: GET_BALANCES,
-})
-
-export const getBalancesSuccess = (balances: Balances): {
-  type: string,
-  payload: {
-    balances: Balances,
-  },
-} => ({
-  type: GET_BALANCES_SUCCESS,
-  payload: {
-    balances,
-  },
-})
-
-export const getBalancesError = (err: Object): {
-  type: string,
-  payload: Object,
-  error: boolean,
-} => ({
-  type: GET_BALANCES_ERROR,
-  payload: err,
-  error: true,
-})
-
-export const setBalancesLoading = (isBalancesLoading: boolean): {
-  type: string,
-  payload: {
-    isBalancesLoading: boolean,
-  },
-} => ({
-  type: SET_BALANCES_LOADING,
-  payload: {
-    isBalancesLoading,
-  },
-})
-
-export const setBalanceByAddress = (address: Address, balance: number): {
-  type: string,
-  payload: {
-    address: Address,
-    balance: number,
-  },
-} => ({
-  type: SET_BALANCE_BY_ADDRESS,
-  payload: {
-    address,
-    balance,
-  },
-})
+export const SET_INITIAL_ITEMS = '@@digitalAssets/SET_INITIAL_ITEMS'
+export const ADD_CUSTOM_ASSET = '@@digitalAssets/ADD_CUSTOM_ASSET'
+export const REMOVE_ASSET = '@@digitalAssets/REMOVE_ASSET'
+export const UPDATE_ASSET = '@@digitalAssets/UPDATE_ASSET'
+
+export function setInitialItems(items: DigitalAssets) {
+  return {
+    type: SET_INITIAL_ITEMS,
+    payload: {
+      items,
+    },
+  }
+}
 
-export const setBalanceByAddressSuccess = (balances: Balances): {
-  type: string,
-  payload: {
-    balances: Balances,
-  },
-} => ({
-  type: SET_BALANCE_BY_ADDRESS_SUCCESS,
-  payload: {
-    balances,
-  },
-})
+export function addCustomAsset(address: Address, name: string, symbol: string, decimals: number) {
```

Comment:
I suggest to use `address: Address, name: string, symbol: string, decimals: number`
as `payload: DigitalAssetBaseFields`

Reply:
Will be fixed in the next PR, we need to add NetworkId here